SerialPortStream-netstandard15.sln

```diff
@@ -1,22 +1,28 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26430.13
-MinimumVisualStudioVersion = 10.0.40219.1
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "netstandard", "code\SerialPortStream-netstandard15.csproj", "{104EE9DA-BCC0-4DB9-AA27-617544FCA2E2}"
-EndProject
-Global
-	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Debug|Any CPU = Debug|Any CPU
-		Release|Any CPU = Release|Any CPU
-	EndGlobalSection
-	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{104EE9DA-BCC0-4DB9-AA27-617544FCA2E2}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{104EE9DA-BCC0-4DB9-AA27-617544FCA2E2}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{104EE9DA-BCC0-4DB9-AA27-617544FCA2E2}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{104EE9DA-BCC0-4DB9-AA27-617544FCA2E2}.Release|Any CPU.Build.0 = Release|Any CPU
-	EndGlobalSection
-	GlobalSection(SolutionProperties) = preSolution
-		HideSolutionNode = FALSE
-	EndGlobalSection
-EndGlobal
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.26430.13
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SerialPortStream-netstandard15", "code\SerialPortStream-netstandard15.csproj", "{104EE9DA-BCC0-4DB9-AA27-617544FCA2E2}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+		Signed_Release|Any CPU = Signed_Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{104EE9DA-BCC0-4DB9-AA27-617544FCA2E2}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{104EE9DA-BCC0-4DB9-AA27-617544FCA2E2}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{104EE9DA-BCC0-4DB9-AA27-617544FCA2E2}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{104EE9DA-BCC0-4DB9-AA27-617544FCA2E2}.Release|Any CPU.Build.0 = Release|Any CPU
+		{104EE9DA-BCC0-4DB9-AA27-617544FCA2E2}.Signed_Release|Any CPU.ActiveCfg = Signed_Release|Any CPU
+		{104EE9DA-BCC0-4DB9-AA27-617544FCA2E2}.Signed_Release|Any CPU.Build.0 = Signed_Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {377CBCBE-D342-42CD-A5CE-3998AB537ECD}

```

Comment:
Do you know what this change is for?

Reply:
Seems as VS is creating that automatically. See also https://stackoverflow.com/questions/45861679/what-is-the-new-globalsection-in-a-vs2017-15-3-solution-file/45862755.

I did an additional test. I removed the section, opened the solution with the VS experimental instance that has no extensions installed/active, modified the solution and then compared the differences. As soon as I modify the solution VS adds the section (this time with a different GUID) to the solution file.